README.md

```diff
@@ -4,7 +4,7 @@
 
 <sup>**Social Media Photo by [JJ Ying](https://unsplash.com/@jjying) on [Unsplash](https://unsplash.com/)**</sup>
 
-### This is not a crawler!
+### This is ~~not a~~ better than a crawler! :3
```

Comment:
I'd rather remove the sentence instead of making it pretentious as crawling is not the main goal of this project.